benefits/enrollment/views.py

```diff
@@ -45,8 +46,20 @@ def token(request):
             audience=payment_processor.audience,
         )
         client.oauth.ensure_active_token(client.token)
-        response = client.request_card_tokenization_access()
-        session.update(request, enrollment_token=response.get("access_token"), enrollment_token_exp=response.get("expires_at"))
+
+        try:
+            response = client.request_card_tokenization_access()
+        except Exception as e:
+            if isinstance(e, HTTPError) and e.response.status_code >= 500:
+                sentry_sdk.capture_exception(e)
```

Comment:
Send a new specific analytic event for us failing to get an access token for card tokenization

Reply:
See [line 53](https://github.com/cal-itp/benefits/pull/2089/files#diff-b67985b5acabad774a30a1fccbb912e611d4056d38316cd6614ce601f6930335R53)